### PR TITLE
[WIP]Extended mcp wait time per node from 20 to 25

### PR DESCRIPTION
**Pull request by JianLi-RH**

### CODE REVIEW

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-4.13-upgrade-from-stable-4.13-azure-ipi-fullyprivate-proxy-p1-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-4.13-upgrade-from-stable-4.13-azure-ipi-fullyprivate-proxy-p1-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-4.13-upgrade-from-stable-4.13-azure-ipi-fullyprivate-proxy-p1-f28

File: ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh [hunk 1]
```diff
@@ -359,6 +359,7 @@ function wait_machineconfig_applied() {
     echo "MCP ${role} status is ${mcp_status}"
     if [[ X"$mcp_status" != X"True" ]]; then
         echo "Timeout waiting for mcp updated"
+        oc describe mcp ${role}
```
Comment (shellyyang1989) [inline ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh:362]: https://github.com/openshift/release/blob/aecef90e8e2bfb35c74fa0b1034d4d03cbbd2aba/ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh#L17-L18 does describe the mcp.

**Pull request closed without merging**